vmdb/app/controllers/api_controller/policy_actions.rb

```diff
@@ -0,0 +1,12 @@
+class ApiController
+  module PolicyActions
+    #
+    # Policy Actions Subcollection Supporting Methods
+    #
+
+    def policy_actions_query_resource(object)
+      return {} unless object && object.respond_to?(:miq_actions)
```

Comment:
Can't this be simplified to `return {} unless object.respond_to?(:miq_actions)` because if `object` is `nil`, it will not `respond_to?(:miq_actions)`.